src/classes/services/CaseService.cls

```diff
@@ -60,4 +65,57 @@ public inherited sharing class CaseService {
         update casesToUpdate;
         insert appointmentResources;
     }
+
+    public static void createTimeSlot(List<Case> newCases) {
+
+        List<Time_Slot__c> timeSlots = new List<Time_Slot__c>();
+
+        Map<String, Operating_Hours__c> operatingHoursByIdsAndDates = new Map<String, Operating_Hours__c>();
```

Comment:
MINOR
An alternate map could be:
` Map<Id, Map<Date, OperatingHours__c>> operatingHoursByIdsByBoutiqueIds` - more obvious